src/components/widgets/filesystem/FileSystem.vue

```diff
@@ -435,7 +507,10 @@ export default class FileSystem extends Mixins(StateMixin, FilesMixin, ServicesM
       } else {
         if (
           e.type === 'click' && item.type === 'file' &&
-          this.$store.state.config.uiSettings.editor.autoEditExtensions.includes(`.${item.extension}`)
+          (
+            this.$store.state.config.uiSettings.editor.autoEditExtensions.includes(`.${item.extension}`) ||
+            this.timelapseBrowser
```

Comment:
Why not leave the default behavior here (follow the settings) instead of forcing auto-open?